shared/.pronto_spell.pws

```diff
@@ -425,6 +426,7 @@ sunday
 sunduk
 symbian
 tabindex
+talkable
```

Comment:
This should not be needed because it is covered by regular expression.
If it is not working for some reason, that should be investigated instead of whitelisting *able words.
https://github.com/talkable/talkable-style/blob/93c5f6acae435a646248374c8627ca48594460e9/shared/.pronto_spell.yml#L5

Reply:
Sorry, didn't notice it. Seems like my local config was outdated and now it works.